WSLBlobNFS/WSLBlobNFS.psm1

```diff
@@ -321,6 +324,13 @@ function Initialize-WSLBlobNFS-Internal
             Write-Verbose "NFS & Samba are not installed. Installing NFS & Samba."
         }
         Invoke-WSL "'$modulePathForLinux/$wslScriptName' installnfssmb $smbUserName"
+        if ($LastExitCode -ne 0)
+        {
+            Write-Error "Installing NFS & Samba failed. Run 'Initialize-WSLBlobNFS -Force' to correctly install NFS & Samba."
```

Comment:
Let's talk about this, but this looks like a confusing message to user.. how can Force fix things?